spec/index.bs

```diff
@@ -3335,6 +3335,20 @@ GPURenderPipeline includes GPUObjectBase;
 GPURenderPipeline includes GPUPipelineBase;
 </script>
 
+{{GPURenderPipeline}} has the following internal slots:
+
+<dl dfn-type=attribute dfn-for=GPURenderPipeline>
+    : <dfn>\[[descriptor]]</dfn>, of type {{GPURenderPipelineDescriptor}}
+    ::
+        The {{GPURenderPipelineDescriptor}} describing this pipeline.
+
+        All optional fields of {{GPURenderPipelineDescriptor}} are defined.
+
+    : <dfn>\[[strip_index_format]]</dfn>, of type {{GPUIndexFormat}}?
+    ::
+        The format index data this pipeline requires, initially `null`.
```

Comment:
nit: `undefined` instead of null maybe to match the descriptor?

Reply:
Done!